garage/envs/mujoco/hopper_env.py

```diff
@@ -3,8 +3,8 @@
 from garage.core import Serializable
 from garage.envs import Step
 from garage.envs.mujoco import MujocoEnv
+from garage.logger import tabular
```

Comment:
AFAICT there is no call to `logger.log` in here

Reply:
The file doesn't use the logger but the `log diagnostics` method logs to the tabular.

Reply:
The way it was implemented it was the responsibility of the model to dump the tabular, so I tried to keep it that way. Is that the right way to go?

Reply:
hmm i see. you're right. this is more a design flaw of the env than anything (implicitly sharing global data)